common/utils.js

```diff
@@ -2,12 +2,25 @@
 
 import {numpy} from './libs/numpy.js';
 
-function sizeOfShape(shape) {
+export function sizeOfShape(shape) {
   return shape.reduce((a, b) => {
     return a * b;
   });
 }
 
+export async function getArrayBuffer(url) {
```

Comment:
How about name it as `getBufferFromUrl`?

Reply:
BTW, could you please add annotations to explain why refactor this method and where it will be used (ES6 env, node env...) ? And please help replace all `fetch()` in webnn-samples, maybe you can do it in a separated PR.

Reply:
I have updated codes, PTAL, thanks! @Honry